src/spatial/models/PolygonSensorOptimizer.py

```
@@ -92,7 +92,8 @@
             if isinstance(features, gpd.GeoDataFrame):
                 self.water_features = features
         except Exception as e:
-            print(f"Warning: Failed to fetch water features: {e}")
+#            print(f"Warning: Failed to fetch water features: {e}")
+            print(f"Warning: Failed to fetch water features: ")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)